vignettes/adrs.Rmd

```diff
@@ -670,41 +730,128 @@ dataset_vignette(
 )
 ```
 
-## Derive Parameters using Independent Review Facility (IRF)/Blinded Independent Central Review (BICR) responses {#irf}
+## Derive Non-standard Parameters {#nonstandard}
+
+As `admiral::derive_extreme_event()` is very flexible, it is easy to implement
+non-standard parameters. Below two examples for modified RECIST 1.1 parameters
+are shown.
+
+### Adding a Criterion for Confirmed Clinical Benefit
+Confirmed clinical benefit was defined before as confirmed response or CR, PR,
+SD, or NON-CR/NON-PD at least 42 days after randomization. Here an alternative
+definition is implemented which considers PD more than 42 days after
+randomization as an additional criterion for clinical benefit.
+```{r}
+adrs <- adrs %>%
+  derive_extreme_event(
+    by_vars = exprs(STUDYID, USUBJID),
+    order = exprs(desc(AVALC), ADT, PARAMCD),
+    mode = "first",
+    events = list(
+      event(
+        condition = PARAMCD == "CRSP" & AVALC == "Y",
+        set_values_to = exprs(AVALC = "Y")
+      ),
+      cb_y,
+      event(
```

Comment:
good example! why don't we create this as an object first in the example and then feed it in to the function call? might be easier for users to follow and then its more consistent to how you build the other parameter derivations

Reply:
Updated